docs/api/commands/create.md

```diff
@@ -161,6 +161,10 @@ Triton's secure, multi-tenant, container-native environment imposes some differe
 * `-v`, `--volume` and `--volumes-from` behave slightly differently in a Triton's container-native environment. See [volumes](../features/volumes.md).
 * `--volume-driver` and other `docker volume` commands are unimplemented at this time.
 
+Docker Swarm's affinity filters for controlling on which server a container
```

Comment:
@tgross I consistently wrap at <80 columns. ;)  That, IMO, helps with developers reading/editing the raw Markdown docs; and with diffs.  I find the crazy-long lines frustrating to edit.

It *does* require the author to be using an editor that can do paragraph reflow.